gogs_client/interface.py

```diff
@@ -1,12 +1,12 @@
 import requests
 
 from gogs_client._implementation.http_utils import RelativeHttpRequestor, append_url
-from gogs_client.entities import GogsUser, GogsRepo
+from gogs_client.entities import GogsUser, GogsRepo, TokenInfo
 
 
 class GogsApi(object):
     """
-    A Gogs client, serving as a wrapper around the Gogs HTTP API.
+    A Gogs API entries, serving as a wrapper around the Gogs HTTP API.
```

Comment:
Why does this need to change? `API entries` is a little misleading, since there isn't a one-to-one correspondence between these functions and the Gogs API endpoints.